mod_upload/forms.py

```diff
@@ -43,6 +43,12 @@ class CommonSampleForm(Form):
     version = SelectField(
         'Version', [DataRequired(message='Version is not selected')],
         coerce=int)
+    report = SelectField('Do you want to report Issue on GitHub?', choices=[
+                         ('n', 'No'), ('y', 'Yes')])
+    IssueTitle = TextAreaField(
+        'Issue Title', [DataRequired(message='Title is not filled in')])
```

Comment:
DataRequired might cause trouble when submitting the form and not submitting an issue.